docs/framework/wcf/feature-details/using-the-discovery-client-channel.md

```diff
@@ -28,13 +28,13 @@ When writing a WCF client application you need to know the endpoint address of t
   
 1.  <xref:System.ServiceModel.Discovery.DiscoveryClientBindingElement.FindCriteria%2A>, which is used to describe the service you want to call.  
   
-2.  <!--zz <xref:System.ServiceModel.Discovery.DiscoveryClientBindingElement.DiscoveryEndpoint%2A>  --> `DiscoveryEndpoint`, which specifies the discovery endpoint to send discovery messages to.  
+2.  xref:System.ServiceModel.Discovery.DiscoveryClientBindingElement.DiscoveryEndpoint%2A> which specifies the discovery endpoint to send discovery messages to.  
```

Comment:
missing angle bracket for the xref link. Should probably be System.ServiceModel.Discovery.DiscoveryClientBindingElement.DiscoveryEndpointProvider instead

Reply:
I read the styling guide, but I couldn't figure out when to use the * (%2A) symbol, so I kept those in there.

Reply:
You can keep the %2A. It was more about the name of property that is different. Sorry for not being clear. Usually for types we don't use %2A or when we're talking about a specific API. For things that can have multiple variations like method or constructors and we want to refer to that in a generic way, then we use the %2A